src/components/features/ProductSearch/ProductSearch.js

```diff
@@ -10,9 +10,32 @@ const ProductSearch = () => (
   <form action='' className={styles.root}>
     <div className={styles.category}>
       <FontAwesomeIcon className={styles.icon} icon={faListUl} />
-      <select name='' id=''>
-        <option value=''>Select a category</option>
-      </select>
+      <ul className={styles.list}>
+        <li className={styles.listCategory}>
+          <span value=''>Select a category</span>
+        </li>
+        <li className={styles.activeList}>
```

Comment:
Nazwa `activeList` sugeruje, ze jest to aktywna lista a tak nie jest, jest to element listy i napewnoi nie jest aktywny bo jest schowany na wejściu na stronę. Proponuję popracować nad nazwą

Reply:
poprawione